### PR TITLE
Issue26 - Better mobile support 

### PR DESCRIPTION
**Pull request by Nizzypal**
Addresses #26 
### All Submissions

image
image
image
image

### New Feature Submissions

* [ ] Does this relate to an existing issue, or has it been talked about prior to being done (if a major change)?

### Notes

Please note, your code must pass all tests and lint checks before it can be merged.

### PR Description

Better mobile support

### CODE REVIEW

File: frontend/src/views/Combat.vue
Comment (seiyria) [inline frontend/src/views/Combat.vue:85]: please fix this indentation

File: frontend/src/components/smart/WeaponGrid.vue
Comment (seiyria) [inline frontend/src/components/smart/WeaponGrid.vue:298]: do we really need a media query? can this be done with bootstrap?

File: frontend/src/components/BigButton.vue
Comment (seiyria) [inline frontend/src/components/BigButton.vue:56]: do we need a media query? can this be done with bootstrap?

File: frontend/src/views/Combat.vue
Comment (seiyria) [inline frontend/src/views/Combat.vue:460]: same comments as above re: media queries

**Review by seiyria — CHANGES_REQUESTED** [review]

File: frontend/src/views/Combat.vue
Comment (Nizzypal) [reply frontend/src/views/Combat.vue:85]: Got it

File: frontend/src/components/smart/WeaponGrid.vue
Comment (Nizzypal) [reply frontend/src/components/smart/WeaponGrid.vue:298]: Bootstrap's functionality is mainly the responsiveness of the elements,  mainly implemented thru cols which are mainly setting the width of divs. I've already tried to not do media queries since you didnt like them but these changes are not about arrangement but about the properties of individual elements which Bootstrap as far as I know don't manipulate directly.

File: frontend/src/components/BigButton.vue
Comment (Nizzypal) [reply frontend/src/components/BigButton.vue:56]: As above.

File: frontend/src/views/Combat.vue
Comment (Nizzypal) [reply frontend/src/views/Combat.vue:460]: As above.

File: frontend/src/components/smart/WeaponGrid.vue
Comment (Nizzypal) [reply frontend/src/components/smart/WeaponGrid.vue:298]: If you want we can use JQuery to reassign classes to the  individual elements.  We will have to load a bunch of  JS for  each affected element.   One caveat to is since we will have to check screen size at some point (whether from an event handler or at  the beginning) we will have to deal with as I'm sure you know  is the JS issue of always having to check if elements exist when they are first loaded.  I'm also not  sure  of the  performance impact of  that much JS might have...

Comment (seiyria) [reply frontend/src/components/smart/WeaponGrid.vue:298]: jquery is off the table.

**Comment by seiyria** [comment]
when posting a PR, you need to follow the template. please post screenshots in your original post and mark what issue it relates to for easy cross referencing.

File: frontend/src/components/smart/WeaponGrid.vue
Comment (Nizzypal) [reply frontend/src/components/smart/WeaponGrid.vue:298]: ok...

**Comment by Nizzypal** [comment]
> when posting a PR, you need to follow the template. please post screenshots in your original post and mark what issue it relates to for easy cross referencing.

I posted a link to the issue above (it says Addresses: 26 - I used the link funcition here so that  you can click on it to take  you to  the original ticket)
Also where can I find the template you want me to follow?

**Comment by seiyria** [comment]
Every time you open a PR there's a template you should follow: 
image

**Comment by Nizzypal** [comment]
> Every time you open a PR there's a template you should follow:
> image

oh i thought those were github generated.  Sure tho there are already screenshots of how the solution would  look like  in the  original ticket, you'd like them here too?  As for PR description , I thought i was arleady doing that in the title but if you want it here too ill put it.

**Comment by seiyria** [comment]
They are generated by github, because I made it so any time a PR comes in, the text is there for the opener to fill out. All PR related things should be here in the PR, ideally in the original comment so it can be easily referenced.

**Comment by Nizzypal** [comment]
yup I understand dude.  Just didnt think it  came from u since i didnt see it on other PRs...

**Comment by Nizzypal** [comment]
Hey @seiyria what else needs to be  done here for it to be merged?

**Comment by seiyria** [comment]
@Nizzypal there are merge conflicts to resolve

**Comment by Nizzypal** [comment]
Yeah I mean it had none when we last talked but then another code pull request taht came after this got  merged so...

**Comment by seiyria** [comment]
image

I can't hit the merge button unless the conflicts are resolved.

**Comment by Nizzypal** [comment]
As I mentioned before there was no conflict when we last talked, there is now coz it sat here... that's why I asked coz another conflict might arise again.   
Resolved  it.

**Comment by seiyria** [comment]
yes, and not everything gets merged in the order it's pushed. sometimes you have to resolve merge conflicts.

**Review by seiyria — APPROVED** [review]

**Pull request merged**
